src/shared/src/FlowEventSync.ts

```diff
@@ -0,0 +1,100 @@
+import stringify from 'safe-stable-stringify';
+
+// TODO replace this with the class from the actual source once the repo is updated
```

Comment:
What does this TODO refer to?

Reply:
Got most of the code from the NXi package and wanted to replace it with the TS sources once they updated the repo. But not that relevant anymore as I adapted some parts already. WIll remove